backend/app/controllers/spree/admin/variants_including_master_controller.rb

```diff
@@ -1,6 +1,7 @@
 module Spree
   module Admin
     class VariantsIncludingMasterController < VariantsController
+      belongs_to 'spree/product', find_by: :slug
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.